gapis/gfxapi/synchronization/synchronized_api.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package synchronization
```

Comment:
Please also add a `doc.go`, or rename this to `sync[hronization].go`, and add a package description. Please refer [to this](https://blog.golang.org/godoc-documenting-go-code) regarding how we try to document our packages. We try to have a short one liner starting `// Package xxx ...` followed with a blank commented line and some more meaningful documentation. There's some good and bad examples [in our generated godocs](https://godoc.org/github.com/google/gapid).